flake.nix

```diff
@@ -171,6 +174,20 @@
               source ${writeScript ./nix/shell.sh}
             '';
           });
+        devShells.awslc-fips = devShells.default.overrideAttrs
+          (finalAttrs: previousAttrs: {
+            # Re-include cmake to update the environment with a new libcrypto.
+            buildInputs = [ pkgs.cmake aws-lc-fips ];
+            S2N_LIBCRYPTO = "awslc-fips";
+            # Integ s_client/server tests expect openssl 1.1.1.
```

Comment:
These next couple of changes seem unrelated to adding a new libcrypto? What's happening here?

Reply:
iirc, the shellHook couldn't be appended to, so in order to change one value(PS1), the whole thing had to be redefined. There is one irreverent comment on 183 I'll remove though..